src/main/resources/dk/sdu/se_f22/contentmodule/management/PostgresScript.txt

```diff
@@ -0,0 +1,8 @@
+create table pages
+(
+    id        serial
+        constraint pages_pk
+            primary key,
+    html      text      not null,
+    timestamp timestamp not null
```

Comment:
What is this?
Please, if you are needing to add a table to the database, then use the migration system

Reply:
sorry babe 😔